.travis.yml

```diff
@@ -28,3 +27,4 @@ install:
 
 script:
   - vendor/bin/phpspec run
+  - vendor/bin/phpunit tests
```

Comment:
If not configured in phpunit.xml then PHPUnit expects a `phpunit [options] <directory>`. I'll add the directory to the XML.